Add WABT to docker image

geky opened the pull request:
The WebAssembly binary toolkit (WABT) has several tools that would help with development on .wasm files

Note: I'm not aware of any apt package for this, so we need to compile from source. The adds ninja as a requirement, though @mathias-arm mentioned we will already need ninja for https://github.com/veracruz-project/veracruz/pull/367.

resolves https://github.com/veracruz-project/veracruz/issues/382

I'm going to look into seeing if we can remove the wasm-checker specific copy of WABT and use this per @dreemkiller's comments

geky committed:
Add WABT to docker image

The WebAssembly binary toolkit (WABT) has several tools that would help
with development on .wasm files

mathias-arm commented:
We probably clone a specific branch / tag of the repository. Only the files in `bin` are installed `wasm-checker` would require the header and library files. Could we leverage the `install` target in the `Makefile` to install everything?

geky commented:
Ah, I didn't catch that they had an `install` target. It looks like it worked so I've updated the PR. I've also changed the checkout to specify v1.0.27 which is the current version of wabt. Though we should probably update this anytime we adopt new wasm features.

I also found the difference in `bin`/`wasm-checker` requirements and mentioned them here:
https://github.com/veracruz-project/veracruz/issues/382

I think it may be better to just have two copies of wabt for these.
